docs/tutorials/gettingstartedexample/gse-create-tdms.md

```diff
@@ -0,0 +1,193 @@
+To control the interpacket delay set the **minimum delay between packets (cycles)** and **line blanking (cycles)**.
+> Note: If you set the interpacket delay too low (which is an arbitrary level based on data path delays and serializer specifications) you will run into packet timing errors when you try to run the data from the Generation Example. If you set **number of frames** very high it will take a very long time to iterate on configuration settings. It does not calculate actual fps until after the TDMS files have been generated. If you create a massive file you can run out of disk space and not get a valid data set.
+    
+If the interpacket delay and packet duration of the packets is greater than the desired frame period, the generated output files will not achieve the desired frame rate. The VI will update the **Actual Frame Rate (fps)** indicator after creating the TDMS files to let the user see the final frame rate achieved.
+
+The interpacket delay in a generated TDMS file is largely goverened by the value of **minimum delay between packets (cycles)** and **line blanking (cycles)** and the rules defined below:
+- Min Delay = **minimum delay between packets (cycles)**		
+- Max Delay = the greater of **minimum delay between packets (cycles)** and **line blanking (cycles)**
+- Long Packet following a Long Packet has Max Delay
+- Line End or Frame End following a long packet has Min Delay
+- Any packet following a Frame Start, Frame End, or Line Start has the Min Delay
+- Line Start following a Line End has Max Delay
+- Frame End following Line End has Min Delay
+    
+The rules above can be summarized to:
+
+| Current Packet | Next Packet | Delay (cycles) between end of Current and start of Next Packet |
+|-|-|-|
+| Frame Start | Line Start | Min Delay |
+| Frame Start | Long Packet | Min Delay |
+| Frame End | Frame Start | Determined by desired frame rate, but must at least equal Min Delay |
+| Line Start | Long Packet | Min Delay |
+| Line End | Line Start | Max Delay |
+| Line End | Frame End | Min Delay |
+| Long Packet | Long Packet | Max Delay |
+| Long Packet | Line End | Min Delay |
+| Long Packet | Frame End | Min Delay |
+    
+    
+The figure below shows what the timing of the generated LLP packets will look like with **include line sync packets** set to false.
+   ![No Line Sync Packets Diagram](../../images/PXIe-148X-CreateTDMS-NoLineSyncPackets-Diagram.png)
+
+The figure below shows what the timing of the generated LLP packets will look like with **include line sync packets** set to true.
+   ![Line Sync Packets Diagram](../../images/PXIe-148X-CreateTDMS-LineSyncPackets-Diagram.png)
+    
+    
+The following scenarios will illustrate the above concepts in more detail.
+        
+## Determining the maximum frame rate for a given frame size    
+> Note: For the purposes of this tutorial, all input control values not specified should be left as the default value.
```

Comment:
Add a short description between the Title and the note that describes what this section is doing.  For example:

The Create CSI-2 Packet TDMS Files VI can be used to easily determine a maximum theoretical frame rate for a given frame size. Because the frame rate shown in the **Actual Frame Rate (fps)** indicator is influenced by the Frame Data Configuration and Frame Timing Configuration controls, this section walks through how to adjust specific control to ensure that the returned value matches the theoretical maximum for a given frame size.